ui/packages/catalog-ui-search/src/main/webapp/react-component/container/metacard-interactions/export-interaction.tsx

```diff
@@ -8,6 +8,10 @@ const store = require('../../../js/store.js')
 const lightboxInstance = require('../../../component/lightbox/lightbox.view.instance.js')
 
 const onExport = (props: Props) => {
+  const result = props.model.models[0]
+  store.clearSelectedResults()
```

Comment:
❓ Could you readd the previously selected results after displaying the lightbox? It doesn't seem like the user would want the other results they have selected to be cleared.

Reply:
To add on to this, I'm confused as to why this file is importing store directly.  That means none of this will work on other routes.